data/issues/emissions/emission_data_calculations.py

```diff
@@ -33,10 +32,35 @@ def get_n_prep_data_from_smhi(df):
 
     # Change the column names to the first rows entries
     df_smhi = df_smhi.rename(columns=df_smhi.iloc[0])
-    df_smhi = df_smhi.drop([0])  # remove row 0
-
-    df_smhi = df_smhi[(df_smhi['Huvudsektor'] == 'Alla') & (df_smhi['Undersektor'] == 'Alla')
-                      & (df_smhi['Län'] != 'Alla') & (df_smhi['Kommun'] != 'Alla')]
+    df_smhi = df_smhi.drop([0])  # remove row 0o
```

Comment:
Här skulle du kunna refaktorera om du vill, genom att bryta ut kod som återkommer till funktioner. Undrar även dicten med sector dataframes skulle kunna mergas in i df_smhi redan i get_n_prep_data_from_smhi så att funktionen inte returnerar båda?

Reply:
Håller med, men jag förstod det som att pythonkoden ändå ska städas upp snart. Då tänker jag att det är bättre att låta det vara för den här ändringens skull. För om jag börjar dra i det kommer det ena leda till det andra och jag kommer börja städa den här filen också...

Reply:
Fattar vad du tänker. Låter bra, får bli del av större refaktorering senare